lib/Driver.js

```diff
@@ -139,9 +138,21 @@ module.exports = class Driver {
    *
    * @returns {string} Returns the key's id.
    */
-  async computeKeyId({key}) {
+  computeKeyId({key}) {
     return `did:key:${key.fingerprint()}#${key.fingerprint()}`;
   }
+
+  /**
+   * Generates a public verification method from a keypair.
+   *
+   * @param {LDKeyPair} key
+   *
+   * @returns {string} Returns the key's id.
+   */
+  getPublicVerificationMethod({key}) {
```

Comment:
```suggestion
  getPublicVerificationMethod({key} = {}) {
```

Reply:
Why? The function doesn't work w/o `key` being passed in. This JS wizardly probably does something different than what I thought it had been doing.